airflow/providers/snowflake/hooks/snowflake.py

```diff
@@ -239,6 +247,8 @@ def run(self, sql: Union[str, list], autocommit: bool = False, parameters: Optio
         :type autocommit: bool
         :param parameters: The parameters to render the SQL query with.
         :type parameters: dict or iterable
+        :param handler: The result handler which is called with the result of each statement.
+        :type handler: dict or iterable
```

Comment:
The docstring doesn't match the typing here.

Reply:
Fixed